CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ if(IMPORT_DEPTH_CAM_FW AND NOT INTERNET_CONNECTION)
     set(IMPORT_DEPTH_CAM_FW OFF)
 endif()
 
+set(BUILD_IPDEV           ON CACHE BOOL "Enable IP device support")
```

Comment:
Lets choose between `BUILD_REMOTE_DEVICE` and `BUILD_NETWORK_DEVICE` and apply to rest of the code

Reply:
BUILD_NETWORK_DEVICE is better. What about BUILD_NET_DEVICE?

Reply:
Done